### PR TITLE
[8.0][ISSUE][sale_quotation_number]Fixing Issues #303

### PR DESCRIPTION
**Pull request by mikevhe18**
Fixing Issues #303

### CODE REVIEW

**Comment by andhit-r** [comment]
@mikevhe18 i tested the functionality:

## Test 1
1. I duplicate one quotation

Expected Behavior:
- Sales quotation duplicated

Result:
- Passed

## Test 2
1. I cancel one of the quotation
2. I click **Create New Copy Of Quotation**

Expected behavior:
- Odoo create new quotation revision

Result:

Warning message showed **Constraint Error: Order Reference and revision must be unique per Company.**

This issue involving interaction between sale_quotation_number and sale_order_revision. You should test the interaction of copy method on both modules.

File: sale_quotation_number/model/sale_order.py [hunk 1]
```diff
@@ -25,12 +25,12 @@
...
     _inherit = "sale.order"
 
-    @api.one
+    @api.multi
     def copy(self, default=None):
         if default is None:
             default = {}
         default['name'] = '/'
-        return super(SaleOrder, self).copy(default=default)
+        return super(SaleOrder, self).copy(default)
```
Comment (pedrobaeza) [inline sale_quotation_number/model/sale_order.py:33]: Why do you remove the keyword argument?

**Comment by AaronHForgeFlow** [comment]
I experienced the same issue

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on local with https://github.com/OCA/sale-workflow/pull/323

**Review by StefanRijnhart — CHANGES_REQUESTED** [review]
:-1: I'm putting my money on https://github.com/OCA/sale-workflow/pull/473/files

**Pull request closed without merging**